src/Indice.Features.Cases.AspNetCore/CasesApiFeatureExtensions.cs

```diff
@@ -215,7 +216,12 @@ public static class CasesApiFeatureExtensions
         services.AddElsa(elsa => {
             elsa.UseEntityFrameworkPersistence(ef => ef.UseSqlServer(configuration.GetConnectionString("WorkflowDb")), false)
                 .AddQuartzTemporalActivities()
-                .AddHttpActivities(configuration.GetSection("Elsa").GetSection("Server").Bind)
+                .AddHttpActivities(http => {
+                    http.HttpEndpointAuthorizationHandlerFactory =
+                        ActivatorUtilities.GetServiceOrCreateInstance<AuthenticationBasedHttpEndpointAuthorizationHandler>;
+                    http.BaseUrl = new Uri(configuration["Elsa:Server:BaseUrl"]);
```

Comment:
Maybe, we should enrich the changelog notes with those app-settings sections ("Elsa:Server:BaseUrl" & "Elsa:Server:BasePath"), so that the consumers of the library won't need to read the internal code?

Reply:
Love the use of `ActivatorUtilities.GetServiceOrCreateInstance` factory. Nice one for elsa

Reply:
@ChristosAsvestopoulos As I read on the left the settings where already there for the other 2 properties since there was a Bind to configuration method. So nothing has changed. It should be on the readme though if it is not already